### PR TITLE
Bug fix.

### PR DESCRIPTION
**Pull request by mayabrandi**
sys.exit used where it shouldnt. Causing the script to stop. Fixed now.

### CODE REVIEW

File: EPPs/copy_UDFs_between_WFs.py [hunk 1]
```diff
@@ -70,9 +73,15 @@ def main(lims, args):
     process = Process(lims, id = args.pid)
     CUDF = CopyUDF(process, args.udfs, args.qcstep, args.target_out)
     CUDF.copy_udfs()
+    warninng = ''
```
Comment (patrikgrenfeldt) [inline EPPs/copy_UDFs_between_WFs.py]: typo, warninng => warning

File: EPPs/copy_UDFs_between_WFs.py [hunk 2]
```diff
@@ -41,11 +41,14 @@ def copy_udfs(self):
...
-            arts = lims.get_artifacts(process_type = self.aggregate_qc_step, 
+            arts = lims.get_artifacts(process_type = self.aggregate_qc_step,
                         sample_name = sample.name)
             ## Getting the most recent aggregate_qc_step
             if not arts:
-                sys.exit('Some samples did not go through aggregate qc step.')
+                self.no_qc_step.append(target_art.id)
+                continue
+
+                #sys.exit('Some samples did not go through aggregate qc step.')
```
Comment (patrikgrenfeldt) [inline EPPs/copy_UDFs_between_WFs.py]: I suggest we remove the old code to keep the code clean and readable, ok?

File: EPPs/copy_UDFs_between_WFs.py [hunk 3]
```diff
@@ -70,9 +73,15 @@ def main(lims, args):
...
     if CUDF.failded_udfs:
-        sys.exit('failed to copy some udfs')
+        warninng += 'Failed to copy some udfs'
+    if CUDF.no_qc_step:
+        warninng += 'Some samples did not go through aggregate qc step: ' + ', '.join(CUDF.no_qc_step)
+
+    if warninng:
+        sys.exit(warninng)
     else:
         print >> sys.stderr, 'UDFs were succsessfully copied!'
```
Comment (patrikgrenfeldt) [inline EPPs/copy_UDFs_between_WFs.py]: This looks really C++ish, in't there a more pythonish way of printing to stderr?

File: EPPs/copy_UDFs_between_WFs.py [hunk 4]
```diff
@@ -70,9 +73,15 @@ def main(lims, args):
...
     CUDF.copy_udfs()
+    warninng = ''
 
     if CUDF.failded_udfs:
-        sys.exit('failed to copy some udfs')
+        warninng += 'Failed to copy some udfs'
+    if CUDF.no_qc_step:
+        warninng += 'Some samples did not go through aggregate qc step: ' + ', '.join(CUDF.no_qc_step)
+
+    if warninng:
```
Comment (patrikgrenfeldt) [inline EPPs/copy_UDFs_between_WFs.py]: A real minor thing, a suggestion to improve the readability; that the "normal"/"desired" case comes first and then all special cases comes after, what do you think about that?

Comment (mayabrandi) [reply EPPs/copy_UDFs_between_WFs.py]: 👍

**Pull request merged**